frame/ddc-staking/src/lib.rs

```diff
@@ -1,6 +1,17 @@
 #![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
```

Comment:
Is it required for benchmarks or for another reason?

Reply:
Put in place to add safety to macro expansion in line with other crates